load_testing/common_flows/flow_helper.py

```diff
@@ -207,13 +219,23 @@ def random_cred(num_users):
     Note that YOU MUST run the rake task to put these users in the DB before using them.
     """
     user_num = randint(0, int(num_users) - 1)
+    while user_num in used_nums:
+        logging.debug(
+            f'{LOG_NAME}: User #{user_num} has already been used. Retrying.')
+        user_num = randint(0, int(num_users) - 1)
+        # import pdb
+        # pdb.set_trace()
```

Comment:
probably want to make sure to remove this before merging

Reply:
yep, thanks for catching that.